src/SprykerEco/Zed/Unzer/Persistence/UnzerRepository.php

```diff
@@ -113,6 +115,8 @@ public function findPaymentUnzerOrderItemByIdSalesOrderItem(int $idSalesOrderIte
     }
 
     /**
+     * @deprecated Please remove usage and use UnzerRepository::findPaymentUnzerTransactionCollectionByCriteria
+     *
      * @param string $paymentId
```

Comment:
If marketplace is not using this anymore we can imho remove it from reader and repository